polaris.shopify.com/content/content/actionable-language.md

```diff
@@ -607,7 +610,7 @@ Use the adjective “OK” when merchants need to confirm they’ve read somethi
 
 <!-- end -->
 
-Use the adjective “Accept” when terms of service require legal confirmation before merchants can continue.
+Use the verb “Accept” when terms of service require legal confirmation before merchants can continue.
 
 <!-- dodont -->
 
```

Comment:
> I was wondering if using Accept also requires us to have a Decline button

I agree. We always include `Decline` as a secondary action in the return request workflow on my team, for example:

<img width="576" alt="17-21-4q2op-302nj" src="https://user-images.githubusercontent.com/43160110/219804902-b6b51c9b-8916-47b8-811d-91480a01437d.png">

However, I think adding recommendations for a `Decline` button opens other questions:

- When do we use "decline" vs. "reject" vs. "dismiss"?
- When do we use "accept" vs. "approve?

Let's keep this PR small and limited to to its changeset: `Recommend using Close instead of OK`. I can create a new issue to discuss recommendations for a Decline button so we can ship incremental improvements 😄